pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -350,6 +350,10 @@ private CommonConstants() {
     public static final String CONFIG_OF_SHUTDOWN_RESOURCE_CHECK_INTERVAL_MS =
         "pinot.server.shutdown.resourceCheckIntervalMs";
     public static final long DEFAULT_SHUTDOWN_RESOURCE_CHECK_INTERVAL_MS = 10_000L;
+    public static final String CONFIG_OF_STARTUP_ENABLE_TEMP_CLEANUP =
+        "pinot.server.startup.enablePeriodicTasks";
```

Comment:
`...enablePeriodicTasks` seems not for tmp cleanup?

Reply:
good catch! this is leftover from when I implemented this as a periodic task rather than running once on startup